os/fs/vfs/fs_stat.c

```diff
@@ -91,10 +96,9 @@ static inline int statpseudo(FAR struct inode *inode, FAR struct stat *buf)
 #if defined(CONFIG_FS_SHM)
 		if (INODE_IS_SHM(inode)) {
 			buf->st_mode | S_IFSHM;
-		} else
-#endif
-		{
 		}
+#endif
+		;
```

Comment:
remove ' ; '

Reply:
Negative, it is needed to end conditional (#ifdef- conditional) if-else blocks above.

Reply:
no, previous is better, to align upper case if considering expandability.

Reply:
Negative, it is needed to end if-else blocks that are conditionally included depending on configuration.